### PR TITLE
[feature-11] Added file with settings

### PR DESCRIPTION
**Pull request by rantanevich**
Following chages were made:
- settings.json file was added
- feature-11 folder was created

### CODE REVIEW

File: feature-11/settings.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "key-1": {
+        "a": 1,
+        "b": 2
+    },
+    "key-2": {
+        "a": 3,
+        "b": 4,
+        "c": 5
+    }
+}
```
Comment (rating199) [inline feature-11/settings.json]: You need to add newline at the end of file.

Comment (rantanevich) [reply feature-11/settings.json]: Thanks for your comment! I'll fix it as soon as possible.

**Review by rating199 — CHANGES_REQUESTED** [review]
All well except omitted newline at the end of file.

**Review by rating199 — APPROVED** [review]
Good job!

**Pull request merged**